src/rules/selector-no-empty/__tests__/index.js

```diff
@@ -0,0 +1,46 @@
+import {
+  messages,
+  ruleName,
+} from ".."
+import rules from "../../../rules"
+import { testRule } from "../../../testUtils"
+
+const rule = rules[ruleName]
+
+testRule(rule, {
+  ruleName,
+  config: [true],
+
+  accept: [ {
+    code: ".a, .b {color: red}",
```

Comment:
Please remove the declarations from within the blocks. The focus of the tests is the selector.